### PR TITLE
API PullReviewComment HTMLPullURL should return the HTMLURL

### LINKED ISSUE #11499: [BUG] [API] ListPullReviewComments: HTMLPullURL is not pull.HTMLURL
as title

example:
Get: `http://localhost:3000/api/v1/repos/test01/Reviews/issues/1`
expect: `http://localhost:3000/test01/Reviews/pulls/1`

### PR DESCRIPTION
**Pull request by zeripath**
Fix #11499

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

**Review by techknowlogick — APPROVED** [review]

**Review by CirnoT — APPROVED** [review]
Maybe we should add `APIPullURL`

**Review by CirnoT — APPROVED** [review]

File: modules/convert/pull_review.go [hunk 1]
```diff
@@ -103,7 +103,7 @@ func ToPullReviewCommentList(review *models.Review, doer *models.User) ([]*api.P
 					OrigCommitID: comment.OldRef,
 					DiffHunk:     patch2diff(comment.Patch),
 					HTMLURL:      comment.HTMLURL(),
-					HTMLPullURL:  review.Issue.APIURL(),
+					HTMLPullURL:  review.Issue.HTMLURL(),
```
Comment (6543) [inline modules/convert/pull_review.go:106]: damed ok this is completly my fault - at least I had an issue with it first

**Review by 6543 — APPROVED** [review]
...

**Comment by techknowlogick** [comment]
ping LG-TM

**Comment by techknowlogick** [comment]
@zeripath please send backport :)

**Pull request merged**